libs/utils/energy.py

```diff
@@ -482,8 +482,8 @@ def report(self, out_dir, out_energy='energy.json'):
                 nrg[key] = float(val)
             channels_stats[channel] = nrg
 
-            logging.info('%14s - %s', 'ACME', self._str(channel))
-            logging.info('%14s - %s', 'ACME', nrg)
+            logging.debug('%14s - %s', 'ACME', self._str(channel))
```

Comment:
Well done, but... for consistency shake, should not we add a similar logging.debugs for the AEP class's report().
It should be dumped from within the loop at line 314...